ngrinder-core/src/main/java/net/grinder/engine/agent/PropertyBuilder.java

```diff
@@ -176,6 +176,7 @@ public String buildJVMArgumentWithoutMemory() {
 		jvmArguments = addParam(jvmArguments, properties.getProperty("grinder.param", ""));
 		jvmArguments = addPythonPathJvmArgument(jvmArguments);
 		jvmArguments = addCustomDns(jvmArguments);
+		jvmArguments = addBasePath(jvmArguments);

```

Comment:
I think this can be located in the line 171 so that it can be only activated in security mode.
And I think the more appropriate method name is... addUserDir